setup.py

```diff
@@ -28,15 +30,6 @@ def main():
         copyfile('scripts/apel-ssm.logrotate', 'conf/apel-ssm')
         copyfile('README.md', 'apel-ssm')
 
-        if not path.exists('/var/log/apel'):
```

Comment:
So this has been replaced by the bit below in `data_files`? Is that the better way to do it?

Reply:
I believe so, it lets `setuptools` worry about the directory creation rather than doing it 'manually'

Reply:
:+1: